src/main/java/Command/AddCommand.java

```diff
@@ -0,0 +1,27 @@
+package Command;
+
+import Ping.Task;
+import Ping.TaskList;
+import Ping.UI;
+import Ping.PingException;
+
+
+/**
+ * This class is used to add a task to the list of tasks
+ */
+public class AddCommand extends Command {
+    private Task task;
+    public AddCommand(Task task) {
+        this.task = task;
+    }
+
+    @Override
+    public String perform(TaskList tasks, UI ui) throws PingException {
+        if (task == null) {
+            throw new PingException("Task cannot be empty");
+        } else {
+        tasks.addTask(task);
+        return ui.addMessage(task, tasks.taskSize());

```

Comment:
Small indentation errors here, happens to the best of us!
```suggestion
            tasks.addTask(task);
            return ui.addMessage(task, tasks.taskSize());

```